thalassa/api.py

```diff
@@ -8,7 +8,7 @@
 import xarray as xr
 from holoviews.operation.datashader import dynspread
 from holoviews.operation.datashader import rasterize
-from holoviews.streams import PointerXY
+from holoviews.streams import PointerXY,DoubleTap
```

Comment:
Hi @pmav99 :  I knew this PEP-8 style, but this one assignment per line is what I really don't like in some cases. Why do people want to make the code long in order to stick to style? Particularly, for some short assignment, why do people prefer several lines instead of one line? Can I keep some of these, which actually makes code compact, easy to maintain, out of my programing experience? If you read a really long code with too much redundancy information, you are actually wasting your memory capacity  (see PEP-8 instruction below). 


A Foolish Consistency is the Hobgoblin of Little Minds

One of Guido's key insights is that code is read much more often than it is written. The guidelines provided here are intended to improve the readability of code and make it consistent across the wide spectrum of Python code. As PEP 20 says, "Readability counts".

A style guide is about consistency. Consistency with this style guide is important. Consistency within a project is more important. Consistency within one module or function is the most important.

However, know when to be inconsistent -- sometimes style guide recommendations just aren't applicable. When in doubt, use your best judgment. Look at other examples and decide what looks best. And don't hesitate to ask!

In particular: do not break backwards compatibility just to comply with this PEP!

Some other good reasons to ignore a particular guideline:

    When applying the guideline would make the code less readable, even for someone who is used to reading code that follows this PEP.
    To be consistent with surrounding code that also breaks it (maybe for historic reasons) -- although this is also an opportunity to clean up someone else's mess (in true XP style).
    Because the code in question predates the introduction of the guideline and there is no other reason to be modifying that code.
    When the code needs to remain compatible with older versions of Python that don't support the feature recommended by the style guide.

Reply:
Dear @wzhengui thank you for quoting PEP8 and your comments. 

If possible, I'd rather avoid a long discussion about style guides where each one of us will state our personal preferences. This is an open source project where we hope to attract more collaborators in the future and therefore it is important to follow the conventions that most other open source projects are using. In this context, following PEP8 is the way to go.